transformations/token_replacement/requirements.txt

```diff
@@ -0,0 +1 @@
+python-Levenshtein>=0.12.1
```

Comment:
I am sorry to be the bad guy here but unfortunately this is a library whose license would not be permissible (GNU GPL). Could you please find another library or add a function to implement levenshtein?

The rest of the changes are already approved by two other reviewers, so should be fine to merge. :)

Reply:
Hi @kaustubhdhole: thank you for your review and helpful feedback! 

And don't worry, these changes require minimal effort :-) I've just replaced [python-Levenshtein](https://pypi.org/project/python-Levenshtein) with [editdistance](https://pypi.org/project/editdistance) (MIT license). I hope it fits now, otherwise please let me know, I found some other alternatives.